### DISCUSSION TITLE
Why do you render Seo component on starter default template?

### DISCUSSION THREAD

**Discussion by tabekg** [gatsbyjs/gatsby#36903] [Help] [answered]
### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues: https://github.com/gatsbyjs/gatsby/issues
- [X] This issue is not a question, feature request, RFC, or anything other than a bug report. Please post those things in GitHub Discussions: https://github.com/gatsbyjs/gatsby/discussions

### Summary

https://github.com/gatsbyjs/gatsby-starter-default/blob/master/src/pages/index.js#L74

### Steps to Resolve this Issue

1.
2.
3.
...

**Comment by LekoArts** [comment, marked as answer]
We forgot to remove it from there, it's fixed now. Thanks for the notice
